src/main/java/org/folio/mappers/FolioToRtacMapper.java

```diff
@@ -15,6 +21,14 @@
 
 public class FolioToRtacMapper {
 
+  private final Logger logger = LogManager.getLogger(getClass());
+  private final boolean fullPeriodicals;
+  private static final List<String> periodicalNames = List.of("journal", "newspaper");
```

Comment:
Is there a better way to handle this besides hardcoding these values here?

Reply:
isn't nature-of-content a controlled vocabulary?  So what happens if a tenant replaces these values, or wants to use different values?  Maybe they remove these and just want a single entry "periodical", or add "magazine", etc.?

Reply:
Thanks for pointing this out. I had no information that was somewhere in tenant settings

Reply:
As it was stated in the requirements in MODRTAC-37, an instance is considered to be a periodical when either:

    mode of issuance"= serial
    or "nature of content" is one of (journal, newspaper).

I think, even if theese values are absent in a tenant, there will be no error, the system just would not be able to tell if an instance is a periodical or not.